client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -110,6 +110,8 @@
   private UrlList baseUrls;
   private SSLSocketFactory sslSocketFactory;
   private BasicAuthCredentialProvider basicAuthCredentialProvider;
+  private int httpConnectTimeoutMS = 2000;
```

Comment:
I'm also in favor of picking sane defaults and not adding configuration parameters yet. 
In this case these two variables should be `private static final` with the appropriate naming format for such variables.

Reply:
Agreed, makes sense, changed and increased.